src/ocean/util/serialize/contiguous/Deserializer.d

```diff
@@ -1100,15 +1088,11 @@ struct Deserializer
 
      **************************************************************************/
 
-    template RejectQualifier ( T )
+    template StripQualifier ( T )
     {
         version (D_Version2)
-            static if (!is(T == typeof(mixin("cast()T.init"))))
-            {
-                pragma(msg, "Warning: ", T,
-                " -- type qualifiers are not supported");
-            }
-        alias T RejectQualifier;
+            mixin("static assert (!is(T == immutable));");
+        alias Unqual!(T) StripQualifier;
```

Comment:
Given that there are more qualifiers than `const` and `immutable`, would it be better to only accept `const` rather than only reject `immutable`?